.github/workflows/tag-and-release.yaml

```diff
@@ -52,3 +52,5 @@ jobs:
         run: |
           uds zarf package create --confirm -a arm64 -o oci://ghcr.io/defenseunicorns/packages
           uds zarf package create --confirm -a amd64 -o oci://ghcr.io/defenseunicorns/packages
+          uds zarf package create --confirm -a arm64 --flavor airgap -o oci://ghcr.io/defenseunicorns/packages
+          uds zarf package create --confirm -a amd64 --flavor airgap -o oci://ghcr.io/defenseunicorns/packages
```

Comment:
nit: we could matrix this at this point.

We also may want to add real "testing" during the release now as well to make sure nothing breaks between PRs and a release.